lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -505,6 +505,60 @@ def test_poly3dcollection_alpha():
     ax.add_collection3d(c2)
 
 
+@mpl3d_image_comparison(['add_collection3d_zs_array.png'])
+def test_add_collection3d_zs_array():
+    theta = np.linspace(-4 * np.pi, 4 * np.pi, 100)
+    z = np.linspace(-2, 2, 100)
+    r = z**2 + 1
+    x = r * np.sin(theta)
+    y = r * np.cos(theta)
+
+    points = np.array([x, y, z]).T.reshape(-1, 1, 3)
```

Comment:
```suggestion
    points = np.column_stack([x, y, z]).reshape(-1, 1, 3)
```

Reply:
Your suggestion is more readable, but (strangely) quite a bit slower for me (~4.5 µs vs. 12.5 µs). If this doesn't matter, I'll apply the change.

Reply:
I can't reproduce such a large difference, but readability seems better than speed for tests.